### PR TITLE
Update GitHub actions

### PR DESCRIPTION
**Pull request by ClementPasteau**
new PR of #5276 to run actions

### CODE REVIEW

**Comment by ClementPasteau** [comment]
looks good, I'll let someone else from the team approve so we can merge it!

File: .github/workflows/issues.yml [hunk 1]
```diff
@@ -3,41 +3,56 @@ on:
...
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          auto-close-rules: |
+            [
+              {
+                "type": "body",
+                "regex": ".*Scroll down to '\\.\\.\\.\\.'.*",
+                "message": "Hi @${issue.user.login}! 👋 This issue was automatically closed because it seems that you have not included any steps to reproduce the bug.\n\nGitHub is a place for the technical development of GDevelop itself - you may want to go on the [forum](https://forum.gdevelop.io/), the Discord chat or [read the documentation](https://wiki.gdevelop.io/gdevelop5/) to learn more about GDevelop. Thanks!"
+              }
+            ]
+      - name: Autoclose known beta 105 web-app update bug
```
Comment (AlexandreSi) [inline .github/workflows/issues.yml:21]: I think this can be removed

File: .github/workflows/issues.yml [hunk 2]
```diff
@@ -3,41 +3,56 @@ on:
...
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          auto-close-rules: |
+            [
+              {
+                "type": "body",
+                "regex": ".*_instance.getRawFloatProperty is not a function.*",
+                "message": "Hi @${issue.user.login}! 👋 This issue was automatically closed as this seems to be a known bug. It can be solved by **closing entirely the web-app and opening it again**. This will allow the web-app to auto-update and the problem should be gone."
+              }
+            ]
+      - name: Autoclose known beta 114 web-app update bug
```
Comment (AlexandreSi) [inline .github/workflows/issues.yml:33]: Same here

**Comment by AlexandreSi** [comment]
There's a warning in the build-storybook CI:
image
I don't really know where it comes from but should we fix it now @st-Wook?

**Comment by st-Wook** [comment]
> There's a warning in the build-storybook CI: image I don't really know where it comes from but should we fix it now @st-Wook?

The warning from configure-aws-credentials@v2 is currently being addressed in [this issue](https://github.com/aws-actions/configure-aws-credentials/issues/680). It appears that it will be fixed in the next version, so there's no need for us to fix it now!

File: .github/workflows/issues.yml [hunk 3]
```diff
@@ -3,45 +3,6 @@ on:
   issues:
     types: [opened]
 jobs:
-  moderate:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Autoclose issues about adding a bug without changing the bug report template
```
Comment (st-Wook) [inline .github/workflows/issues.yml]: Should this part be removed?

**Review by AlexandreSi — APPROVED** [review]
LGTM

**Pull request merged**